### PR TITLE
Bug(cli): code.tar.gz file remains after error occurs

### PR DESCRIPTION
**Pull request by Vaibhav91one**
## What does this PR do?

- This PR mainly implements a Try catch and finally block to the functionCreateDeployment.
- As if anything happens in between the finally block will always execute and will remove the remaining file.

## Test Plan

N/A

## Related PRs and Issues

* https://github.com/appwrite/sdk-for-cli/issues/28

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: lib/commands/functions.js [hunk 1]
```diff
@@ -405,6 +405,7 @@ const functionsCreateDeployment = async ({ functionId, code, activate, entrypoin
     /* @param {string} entrypoint */
     /* @param {string} commands */
 
+    try{
```
Comment (stnguyen90) [inline lib/commands/functions.js:408]: 1. Let's not have such a big try/catch
2. This repo is read-only and the change should be done with the [sdk-generator](https://github.com/appwrite/sdk-generator) repo.

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Comment by stnguyen90** [comment]
I'm going to add the hacktoberfest-accepted label just so you get credit, but we'll probably close this PR since we need the change in the SDK generator repo.

**Comment by gewenyu99** [comment]
Hey there! There were a lot of big PRs during this Hacktoberfest, and we wanted to give everyone ample time to collaborate with our engineering team. If you were able to merge your PRs during October, amazing. If it’s still not merged, don’t worry about it either. Either way, we’ve got your Hacktoberfest swag minted and ready to ship. 

Please comment with your Discord username here so we can contact you about your shipping information to deliver your Hacktoberfest swag.

**Comment by stnguyen90** [comment]
closing as change needs to be done in sdk-generator repo.

**Pull request closed without merging**